collections/_article/godot-developer-fund.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Introducing the new Godot Development Fund"
+excerpt: "A direct way to support the development of Godot."
+categories: ["news"]
+author: Emi
+image: /storage/blog/covers/developer-fund.jpg
+date: 2023-09-12 11:51:10
+---
+
+We are excited to introduce a major improvement in terms of funding the development efforts of the Godot Engine - the [Godot Development Fund](https://fund.godotengine.org)! The goal for the Development Fund is to create a direct way for everyone to help us keep the project alive.
+ 
+
+## Reducing Platform Fees, Amplifying Impact
```

Comment:
I think we normally use "Sentence case" for headings, not "Title Case".
```suggestion
## Reducing platform fees, amplifying impact
```

Reply:
That is correct! Thanks!